docs/okeps/okep-5094-layer2-transit-router.md

```
@@ -0,0 +1,840 @@
+    class stor-ovn_cluster_router portStyle;
+    class rtos-layer2-switch portStyle;
+    class GR-node1 routerStyle;
+    class GR-node2 routerStyle;
+    class GR-node3 routerStyle;
+    class ovn_cluster_router routerStyle;
+    class layer2-switch switchStyle
+    class term termStyle;
+    class node1,node2,node3 nodeStyle;
+```
+From the IPv6 RFC the gateway address configured from router advertisements is a link local address, in this case it will
+be a link local address computed from the subnet address `2010:100:200::1`, again this makes the default gw __independent__ of the
```

markdownlint-cli2 (0.17.2)

Strong style
Expected: asterisk; Actual: underscore

(MD050, strong-style)